Dockerfile

```diff
@@ -14,15 +14,17 @@ COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
+COPY graphql-server/ graphql-server/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
-
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o go-bff-server graphql-server/go-server/main.go
```

Comment:
IMO, we should split this GraphQL server and Arcadia controller into two containers or two Pods.